src/tasks/level10.ts

```diff
@@ -139,7 +144,8 @@ export const GiantQuest: Quest = {
       completed: () => have($item`S.O.C.K.`),
       do: $location`The Penultimate Fantasy Airship`,
       choices: () => {
-        return { 178: 2, 182: have($item`model airship`) ? 1 : 4 };
```

Comment:
Same question here, it feels like this PR may be demanding everyone to own bat wings. Or I'm just out of touch and this code never worked / no longer relevant for all ascenders

Reply:
I needed to update how we prioritize choice NCs for Lack of Random Encounter, which required moving that to a choice adventure script.